README.md

```diff
@@ -1,5 +1,11 @@
 # godot-next
 
+## Disclaimer
+
+Please note that this repository has been moved to `godot-extended-libraries/godot-next`. Please visit there if you would like to track its continued development! I am still the maintainer of it within that organization.
+
+## Description
+
```

Comment:
Please adjust your remote to point to godot-extended-libraries/godot-next rather than my own and then remove this commit from your pull request's branch. This message should only exist on my branch.

Reply:
remote changed and commit removed.